types/postcss-page-break/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for postcss-page-break 3.0
+// Project: https://github.com/shrpne/postcss-page-break#readme
+// Definitions by: Adam Thompson-Sharpe <https://github.com/MysteryBlokHed>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+import { PluginCreator } from "postcss";
+
+declare const pageBreak: PluginCreator<never>;
```

Comment:
`never` seems pretty strong; this shouldn't be some sort of empty options type or anything?

Reply:
I don't know if there's any guidelines on what it should be, but I could change it to an empty object if that makes more sense.

Since the original package never actually does anything with the options, I could also just set the type to `any`?

Reply:
I'm not super familiar, there is one other package on DT which uses never for this, so it's potentially fine. I just wasn't sure what this affected.